sdk/src/main/scala/io/horizen/consensus/ConsensusParamsUtil.scala

```diff
@@ -0,0 +1,53 @@
+package io.horizen.consensus
+import io.horizen.fork.{ConsensusParamsFork, ConsensusParamsForkInfo}
+import sparkz.core.block.Block
+
+
+object ConsensusParamsUtil {
+  private var currentConsensusEpoch: Int = 0
```

Comment:
is much  safer to have this class stateless: 
- let's remove currentConsensusEpoch
-  remove Option from the parameters of: getConsensusSecondsInSlotsPerEpoch and getConsensusSlotsPerEpoch 
- if we want the current value we have to pass the current timestamp: add two other versions of the previous methods that accept a timestamp instead of an epoch

Reply:
Done